CMakeLists.txt

```diff
@@ -1,18 +1,20 @@
 cmake_minimum_required(VERSION 2.8.0)
 
 project(express-cpp)
-set(CMAKE_BUILD_TYPE Debug)
-enable_testing()
+
+option (UNIT_TESTS "Enable Unit Tests" OFF)
 
 set (VERSION_MAJOR 0)
 set (VERSION_MINOR 1)
 set (VERSION_PATCH 0)
 
 set (VERSION_STRING_INSTALL "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
 
-add_definitions(--std=c++11)
+if (NOT BUILD_SHARED_LIBRARIES) 
+  SET(CMAKE_FIND_LIBRARY_SUFFIXES ".a")
+endif() 
 
-find_package(Boost 1.59 REQUIRED COMPONENTS filesystem regex system thread filesystem date_time)
+find_package(Boost 1.59 REQUIRED COMPONENTS regex coroutine context thread filesystem system date_time)
```

Comment:
I just tested again, SimpleWebServer as integrated in express no longer needs coroutine or context. It compiles without them for sure: 
`c++   -dynamiclib -Wl,-headerpad_max_install_names -current_version 0.1.0 -o libexpress-cpp.0.1.0.dylib -install_name /Users/linkineo/Devl/tests/express-cpp/build/libexpress-cpp.0.1.0.dylib CMakeFiles/express-cpp.dir/src/application/application.cpp.o CMakeFiles/express-cpp.dir/src/request/request.cpp.o CMakeFiles/express-cpp.dir/src/response/response.cpp.o /usr/local/lib/libboost_regex-mt.a /usr/local/lib/libboost_thread-mt.a /usr/local/lib/libboost_filesystem-mt.a /usr/local/lib/libboost_system-mt.a /usr/local/lib/libboost_date_time-mt.a`